pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -50,8 +51,9 @@ def solve_NLP_subproblem(solve_data, config):
         rhs = ((0 if c.upper is None else c.upper)
                + (0 if c.lower is None else c.lower))
         sign_adjust = 1 if value(c.upper) is None else -1
-        fix_nlp.tmp_duals[c] = sign_adjust * max(0,
-                sign_adjust * (rhs - value(c.body)))
+        fix_nlp.tmp_duals[c] = sign_adjust * max(
+            0, sign_adjust*(rhs - value(c.body)))
+        pass
         # TODO check sign_adjust
```

Comment:
Can we resolve this `#TODO`? I think I put it there originally because I didn't fully understand which way the sign needed to be, but if we can verify it's correct we can remove it.

Reply:
Since we use `sign_adjust` the same way everywhere I guess the sign is probably correct, but we might think about referring to a little derivation somewhere, since it's not super obvious which way the sign needs to be.